routes/routes.js

```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
+
+const usersRouter = require('./users');
+const cardsRouter = require('./cards');
+
+router.use('/', usersRouter);
+router.use('/', cardsRouter);
+router.use('*', (req, res) => {
+  res.status(404).send({ message: 'Запрашиваемый ресурс не найден' });
```

Comment:
Можно лучше: передавать ошибку в обработчик ошибок.
Подробнее: https://github.com/Atm0Sphere94/project12.2/pull/1/files#r429655303 .
```suggestion
  next(new NotFoundError('Запрашиваемый ресурс не найден'));
```